### PR TITLE
docu: magic numbers in rewards module

### PR DESCRIPTION
**Pull request by Nikolaj-K**
This is in the rewards file and its test file.

* Add documentation lines to hardcoded numbers in rewards section.

From me trying to reverse engineering where those floats like `0.68493150684` came from.

* Propose to write `During` instead of `For` w.r.t. the the reward base in the epoch.

Simply because I was shortly confused if it's the whole reward *for* that epoch, which it's not.

### CODE REVIEW

File: src/main/scala/org/constellation/util/Rewards.scala [hunk 1]
```diff
@@ -37,10 +40,10 @@ object Rewards {
...
     val trustEntropyMap = shannonEntropy(transitiveReputationMatrix, neighborhoodReputationMatrix)
     val distro = rewardDistribution(partitonChart, trustEntropyMap)
-    distro.mapValues(_ * rewardForEpoch(curShapshot))
+    distro.mapValues(_ * rewardDuringEpoch(curShapshot))
   }
 
-  def rewardForEpoch(curShapshot: Int) = curShapshot match {
+  def rewardDuringEpoch(curShapshot: Int) = curShapshot match {
     case num if num >= 0 && num < epochOne  => epochOneRewards
     case num if num >= 438000 && num < epochTwo => epochTwoRewards
```
Comment (Nikolaj-K) [inline src/main/scala/org/constellation/util/Rewards.scala]: I'd also replace those hardcoded ints here. I suppose they are a leftover from when the consts like `epochOne` wheren't defined yet?

File: src/test/scala/org/constellation/util/RewardsTest.scala [hunk 2]
```diff
@@ -25,11 +25,11 @@ class RewardsTest extends FlatSpec {
   val thing = r.nextInt(100).toString
   neighborhoodReputationMatrix.updated(thing, 0.0)//Ensure perfect behavior doesn't throw Nan
 
-  "rewardForEpoch" should "return correct $DAG ammount" in {
```
Comment (Nikolaj-K) [inline src/test/scala/org/constellation/util/RewardsTest.scala]: Just see there's a typo here (amount) - I'll add this

File: src/main/scala/org/constellation/util/Rewards.scala (hunk 1 above)
Comment (buckysballs) [reply src/main/scala/org/constellation/util/Rewards.scala]: Sure that would be great thanks!

File: src/test/scala/org/constellation/util/RewardsTest.scala (hunk 2 above)
Comment (Nikolaj-K) [reply src/test/scala/org/constellation/util/RewardsTest.scala]: done

File: src/main/scala/org/constellation/util/Rewards.scala (hunk 1 above)
Comment (Nikolaj-K) [reply src/main/scala/org/constellation/util/Rewards.scala]: done

**Comment by Nikolaj-K** [comment]
Changes done.

Btw. I didn't get where the 10k month actually derives from eventually?

**Comment by Nikolaj-K** [comment]
@buckysballs merge?

**Comment by buckysballs** [comment]
Happy to merge, just needs to be merged with dev

**Comment by Nikolaj-K** [comment]
@buckysballs Not sure what you're saying, but for the sake of it I updated the branch to the current dev status now. (In case you meant that the branch was in the meantime behind the dev branch: this wouldn't have been an issue as githubs git merges conservatively and leaves other files in dev alone.)

Did you see the question from 2 weeks ago? Mostly pointing it out to make sure that this hardcoding of 10k is currently intend as I didn't see the number anywhere else.

**Pull request merged**